osclib/util.py

```diff
@@ -159,4 +159,4 @@ def sha1_short(data):
     if isinstance(data, list):
         data = '::'.join(data)
 
-    return hashlib.sha1(data).hexdigest()[:7]
+    return hashlib.sha1(data.encode('utf-8')).hexdigest()[:7]
```

Comment:
this breaks all usages passing bytes into it :(

Reply:
hmm, really makes this function cumbersome to use as most of the data I want to feed into it originates from osc API call and selected via lxml xpath query.

```python
sha1_short(root.xpath('//foo/@attr'))
```

If there is a way to handle that without manually having to encode each entry that would be preferred. I find this whole game rather cumbersome considering it Just Worked (tm) in python 2.

Reply:
my advice: get over it, python3 is reality.

But I fixed the function to work on both meanwhile

Reply:
> my advice: get over it, python3 is reality.

not questioned at all...fix looks good.